can/interfaces/socketcan/socketcan.py

```diff
@@ -595,18 +595,25 @@ def __init__(
         fd: bool = False,
         **kwargs,
     ) -> None:
-        """
+        """Creates a new socketcan bus.
+
+        It setting some socket options fails, an error will be printed but no exception will be thrown.
+        This includes enabling:
+         - that own messages should be received,
+         - CAN-FD frames and
+         - error frames.
+
         :param channel:
-            The can interface name with which to create this bus. An example channel
-            would be 'vcan0' or 'can0'.
+            The can interface name with which to create this bus.
+            An example channel would be 'vcan0' or 'can0'.
             An empty string '' will receive messages from all channels.
             In that case any sent messages must be explicitly addressed to a
             channel using :attr:`can.Message.channel`.
         :param receive_own_messages:
             If transmitted messages should also be received by this bus.
         :param fd:
             If CAN-FD frames should be supported.
-        :param list can_filters:
+        :param List can_filters:
```

Comment:
I guess we might as well add this to the function declaration? Also I think there's a type  for filters in `typechecking.py` that can be extended, if it isn't sufficient.